examples/client-logger/app-insights-logger/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Generated by npm
```

Comment:
Does this specify anything that the root .gitignore doesn't? If not, we may be able to remove it.

Reply:
Removed this file